tests/test-cross-context-dispatch/ccd-tests/src/test/resources/plans/context-ee10-forward-dump.json

```diff
@@ -0,0 +1,29 @@
+{
+  "class": "org.eclipse.jetty.tests.ccd.common.DispatchPlan",
+  "requestStep": {
+    "class": "org.eclipse.jetty.tests.ccd.common.steps.HttpRequestStep",
+    "method": "GET",
+    "requestPath": "/ccd-ee10/redispatch/ee10"
+  },
+  "steps": [
+    {
+      "class": "org.eclipse.jetty.tests.ccd.common.steps.ContextRedispatchStep",
+      "dispatchType": "FORWARD",
+      "contextPath": "/ccd-ee10",
+      "dispatchPath": "/dump/ee10"
+    }
+  ],
+  "expectedEvents": [
+    "Initial plan: context-ee10-forward-dump.json",
+    "DispatchPlanHandler.handle() method=GET path-query=/ccd-ee10/redispatch/ee10",
+    "org.eclipse.jetty.tests.ccd.ee10.CCDServlet.service() dispatcherType=REQUEST method=GET requestUri=/ccd-ee10/redispatch/ee10",
+    "org.eclipse.jetty.tests.ccd.ee10.DumpServlet.service() dispatcherType=FORWARD method=GET requestUri=/ccd-ee10/dump/ee10"
+  ],
+  "expectedProperties": [
```

Comment:
I also want to test that if the `jakarta.servlet.forward.*` or `jakarta.servlet.include.*`  attributes are requested in an `ee9` and above context that originated from an dispatch from an `ee8` context, that the correct values are returned (which would have been `javax.servlet.forward.*` or `javax.servlet.include.*` in the original `ee8` context).

Similarly, if an `ee9` context does a dispatch to an `ee8` context, the `ee8` context should be able to see the dispatch attributes with a prefix of  `javax.servlet` instead of `jakarta.servlet`.

Reply:
Absolutely, those properties are present.

For example, the properties on this one are reported as ...

```
dispatchPlan.event[0]=Initial plan: context-ee10-forward-dump.json
dispatchPlan.event[1]=DispatchPlanHandler.handle() method=GET path-query=/ccd-ee10/redispatch/ee10
dispatchPlan.event[2]=org.eclipse.jetty.tests.ccd.ee10.CCDServlet.service() dispatcherType=REQUEST method=GET requestUri=/ccd-ee10/redispatch/ee10
dispatchPlan.event[3]=org.eclipse.jetty.tests.ccd.ee10.DumpServlet.service() dispatcherType=FORWARD method=GET requestUri=/ccd-ee10/dump/ee10
dispatchPlan.events.count=4
request.attr[jakarta.servlet.forward.context_path]=/ccd-ee10
request.attr[jakarta.servlet.forward.mapping]=ServletPathMapping{mappingMatch=PATH, matchValue=redispatch, pattern=/redispatch/*, servletName=ccd, servletPath=/redispatch, pathInfo=/ee10}
request.attr[jakarta.servlet.forward.path_info]=/ee10
request.attr[jakarta.servlet.forward.query_string]=<null>
request.attr[jakarta.servlet.forward.request_uri]=/ccd-ee10/redispatch/ee10
request.attr[jakarta.servlet.forward.servlet_path]=/redispatch
request.attr[org.eclipse.jetty.server.Request.Cookies]=[]
request.attr[org.eclipse.jetty.server.Request.maxFormContentSize]=200000
request.attr[org.eclipse.jetty.server.Request.maxFormKeys]=1000
request.attr[org.eclipse.jetty.tests.ccd.common.DispatchPlan]=DispatchPlan[id=context-ee10-forward-dump.json]
request.authType=<null>
request.characterEncoding=<null>
request.contentLength=-1
request.contentType=<null>
request.contextPath=/ccd-ee10
request.dispatcherType=FORWARD
request.header[Accept-Encoding]=gzip
request.header[Host]=localhost:33753
request.header[User-Agent]=Jetty/12.0.11-SNAPSHOT
request.header[X-DispatchPlan]=context-ee10-forward-dump.json
request.localAddr=127.0.0.1
request.localName=127.0.0.1
request.localPort=33753
request.locale=en_US
request.method=GET
request.pathInfo=/ee10
request.pathTranslated=<null>
request.protocol=HTTP/1.1
request.queryString=<null>
request.remoteAddr=127.0.0.1
request.remoteHost=127.0.0.1
request.remotePort=45628
request.remoteUser=<null>
request.requestURI=/ccd-ee10/dump/ee10
request.requestURL=http://localhost:33753/ccd-ee10/dump/ee10
request.requestedSessionId=<null>
request.serverPort=33753
request.servletPath=/dump
```